docs/visualizing-your-data/snowplow-web/v1/index.md

```diff
@@ -0,0 +1,338 @@
+
+Install Streamlit:
+```
+pip install streamlit==1.20.0
+```
+
+Install dependencies:
+
+```
+pip install plotly==5.9.0 geopandas==0.12.2 pydeck==0.7.1
+```
+
+Provide credentials: create file called `credentials.json` in the current directory with the following structure:
+```
+{
+  "account": "<snowflake_account>",
+  "user": "<snowflake_username>",
+  "password": "<snowflake_password>",
+  "database": "<database>",
+  "schema": "<schema>"
+}
+```
+
+Run it:
+```
+streamlit run snowplow.py
+```
+&nbsp; &nbsp; or depending on your setup:
+```
+python -m streamlit run snowplow.py
+```
+
+### 2. Run it on Streamlit on Snowflake
+
+#### Step 1: Create a Streamlit Application:
+  1. Log into Snowsight as a user with the CREATE STREAMLIT privilege.
+  2. In the left navigation bar, select Streamlit Apps.
+  3. Select + Streamlit. The Create Streamlit App window opens.
+  4. Enter a name for your app.
+  5. In the Warehouse dropdown, select the warehouse where you want to run your app and execute queries.
+  6. In the App location dropdown, select the database and schema for your app.
+  7. Select Create. The Streamlit in Snowflake editor opens an example Streamlit application in Viewer mode. Viewer mode allows you to see how the Streamlit application appears to users.
+  8. Click Edit to create your application.
+
+#### Step 2: Install required packages:
+  [PUT](https://docs.snowflake.com/en/sql-reference/sql/put) the [environment.yml](/environment.yml) file to the stage location specified by the `STREAMLIT_ROOT_LOCATION` parameter of the Streamlit object:
+  ```
+  $ SELECT STREAMLIT_ROOT_LOCATION FROM <database>.INFORMATION_SCHEMA.STREAMLITS;
+
+  > <database>.<schema>."<appname> (Stage)"
+  ```
+  ```
+  PUT file:///<full_path>/environment.yml '@<database>.<schema>."<appname> (Stage)"' AUTO_COMPRESS=FALSE OVERWRITE=TRUE;
+  ```
+
+#### Step 4: Provide the geojson [file](/countries.geo.json) that contains the country information required to render the choropleth map:
+  ```
+  PUT file:///<full_path>/countries.geo.json '@<database>.<schema>."<appname> (Stage)"' AUTO_COMPRESS=FALSE OVERWRITE=TRUE;
+  ```
+
+#### Step 5: Copy the code from [snowplow.py](/snowplow.py) inside the editor
+
+and Click `Run`
+
+
+### 3. Run it on Streamlit's cloud
+
+Follow the instruction on [https://streamlit.io/cloud](https://streamlit.io/cloud)
+
+</TabItem>
+
+<TabItem value="Preset" label="Preset">
+
+</TabItem>
+
+</Tabs>
+
+## Dashboards
+
+### Acquisition Overview
+
+The **Acquisition Overview** aims at updating you about the website traffic with a breakdown by marketing channels.
+
+There is also a distinction between `New Users` who have not previously had a session on the website (domain_sessionidx=1) and `Existing users` who have a session with domain_sessionidx > 1.
+
```

Comment:
make these code type filters backticked?

Reply:
Sorry this wasn't very clear, I meant the bits like `domain_sessionidx=1` should also be backticked maybe?